spec/rubocop/cop/rspec/pending_without_reason_spec.rb

```diff
@@ -1,132 +1,133 @@
   end
 
-  context 'when pending by pending step without reason ' \
+  context 'when pending by pending/skip step without reason ' \
           'and not inside example' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
         FactoryBot.define do
           factory :task do
             pending
+            skip
+            pending { true }
+            skip { true }
           end
         end
       RUBY
     end
   end
 
-  context 'when skipped by skip step without reason ' \
-          'and not inside example' do
+  context 'when pending/skip with receiver' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
-        FactoryBot.define do
-          factory :task do
-            skip
+        RSpec.describe Foo do
+          it 'does something' do
+            Foo.pending
+            Foo.skip
           end
         end
       RUBY
     end
   end
 
-  context 'when pending with receiver' do
+  context 'when pending is argument of methods' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
-        it 'does something' do
-          Foo.pending
+        RSpec.describe Foo do
+          it 'does something' do
+            expect('foo').to eq pending
+            foo(bar, pending)
+            foo(bar, pending: pending)
+            is_expected.to match_array [foo, pending, bar]
+          end
         end
       RUBY
     end
   end
 
-  context 'when skip with receiver' do
+  context 'when skip is argument of methods' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
-        it 'does something' do
-          Foo.skip
+        RSpec.describe Foo do
+          it 'does something' do
+            expect('foo').to eq skip
+            foo(bar, skip)
+            foo(bar, skip: skip)
+            is_expected.to match_array [foo, skip, bar]
+          end
         end
       RUBY
     end
   end
 
-  context 'when pending is argument of methods' do
+  context 'when pending/skip is argument of methods' do
     it 'registers no offense' do
       expect_no_offenses(<<~RUBY)
-        it 'does something' do
-          expect('foo').to eq pending
-          foo(bar, pending)
-          foo(bar, pending: pending)
-          is_expected.to match_array [foo, pending, bar]
+        RSpec.describe Foo do
+          it 'does something' do
+            list = [skip, pending]
+            foo(list)
+          end
         end
       RUBY
     end
   end
 
-  context 'when skip is argument of methods' do
+  context 'when pending by example method with block' do
     it 'registers no offense' do
+      # Because `ArgumentError` is raised whenblock is given to example method.
       expect_no_offenses(<<~RUBY)
-        it 'does something' do
-          expect('foo').to eq skip
-          foo(bar, skip)
-          foo(bar, skip: skip)
-          is_expected.to match_array [foo, skip, bar]
+        RSpec.describe Foo do
+          pending 'does something' do
+          end
```

Comment:
The reason I ignored this case is that I assumed this would be an ArgumentError case in the first place.
https://github.com/rubocop/rubocop-rspec/pull/1566/files#r1089015507